src/components/ModalPostPayment.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
```

Comment:
Design of this modal needs work I think.

Reply:
3 proposals: 

![Uploading Screenshot 2023-12-15 at 01.54.53.jpg…]()
![Uploading Screenshot 2023-12-15 at 01.54.30.jpg…]()
![Uploading Screenshot 2023-12-15 at 01.50.36.jpg…]()

Reply:
Modal UI updated, please take another look

Reply:
Looks good!